diff --git a/net/dns/dns_transaction.cc b/net/dns/dns_transaction.cc
index c12f1c728de558b236a0144179bfa268da90fb04..9b6c4a8201b052cfdac7d124861a6b9e700d59e0 100644
--- a/net/dns/dns_transaction.cc
+++ b/net/dns/dns_transaction.cc
@@ -1290,6 +1290,7 @@ class DnsTransactionImpl : public DnsTransaction,
     DCHECK(secure_);
 
     size_t doh_server_index = dns_server_iterator_->GetNextAttemptIndex();
+    BRAVE_MAKE_HTTP_ATTEMPT
 
     unsigned attempt_number = attempts_.size();
     ConstructDnsHTTPAttempt(
